README.md

```diff
@@ -60,6 +60,26 @@ See the [issue tracker] if you're unsure where to start, or feel free to reach o
 
 See also: the Kubernetes [community page].
 
+## Why kind?
+
+kind exists because we wanted something we were comfortable using for Kubernetes's own CI to test Kubernetes itself.
+
+ - kind doesn't use hyperkube or patch in binaries to existing images, we ~build a "real" kubernetes release, our images are more or less (soon to be fully) self-contained.
+ - kind supports building kubernetes with make/bash/docker, bazel, or installing from apt, in addition to pre-published builds.
+ - kind is written in go, and can be used as a library, has binary releases
+ - kind is a clean rewrite vs previous attempts such as test-infra/dind
```

Comment:
I think we can probably drop this point too, it has plusses and minuses and suggests other projects are somehow unclean

Reply:
which point? The "clean rewrite"?